domains/fsd.is-cool.dev

```diff
@@ -0,0 +1,22 @@
+{
+    "description": "FSD-IT",
+
+    "domain": "is-cool.dev",
+    "subdomain": "fsd",
+
+    "owner": {
+        "repo": "https://github.com/username/repo",
+        "email": "fin.seibt9@gmail.com"
+    },
+
+    "record": {
+        "A": ["", ""],
```

Comment:
Half of these are empty records.

CNAME cannot be combined with other records.

We no longer support NS records.